internal/store/pod.go

```diff
@@ -196,6 +196,26 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_pod_restart_policy_always",
+			Type: metric.Gauge,
+			Help: "Set to 1 if the pod has restartPolicy Always, 0 otherwise.",
+			GenerateFunc: wrapPodFunc(func(p *v1.Pod) *metric.Family {
+				value := float64(0)
```

Comment:
Can we make this more readable ? I feel the suggestion below would be idiomatic and suggestive.
```suggestion
				var value float64
```